packages/suite/src/components/wallet/WalletLayout/components/CoinjoinAccountDiscoveryProgress/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { H3, Icon, variables } from '@trezor/components';
+import { CoinjoinSummaryHeader } from '@wallet-components';
+import { Card, Translation } from '@suite-components';
+import { AccountLoadingProgress } from './AccountLoadingProgress';
+import { RotatingFacts } from './RotatingFacts';
+
+const Container = styled(Card)`
+    flex-direction: column;
+    align-items: center;
+    padding-top: 36px;
+    padding-bottom: 36px;
+`;
+
+const FactHeading = styled.div`
+    display: flex;
+    align-items: center;
+    color: ${({ theme }) => theme.TYPE_ORANGE};
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-transform: uppercase;
+`;
+
+const SparksIcon = styled(Icon)`
+    margin-right: 4px;
+    padding-bottom: 2px;
+`;
+
+export const CoinjoinAccountDiscoveryProgress = () => {
+    const theme = useTheme();
+
+    return (
+        <>
+            <CoinjoinSummaryHeader />
```

Comment:
I believe CoinjoinSummaryHeader could be removed from here completely (at least anonymity level setup, but that is supposed to be moved from the summary header to Account details page...)

<img width="650" alt="image" src="https://user-images.githubusercontent.com/3729633/200802838-52368b52-a26f-4abb-8986-5ceb10bcf396.png">

Reply:
I wasn't sure wether to remove the lvl selector already or let someone else remove it as a part of some related issue and decided to keep the header the same for consistency.

figma:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/45338719/200816492-df6568f7-9a60-4328-b34a-dc869c082a5f.png">

Reply:
Do you want me to remove the anonymity lvl selector in this PR or remove it everywhere later?

Reply:
let's keep it for now